### ISSUE TITLE
Date admin widget

### ISSUE THREAD

**Issue by sverhoeven** [research-software-directory/research-software-directory#444]
In the project edit page you have to fill the start and end date, now this is done with a text field and "^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$" as regular expression.  It would be nice if the text field could be replaced with a date picker like https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

**Comment by jspaaks** [comment]
Agree. In the meantime, maybe we can simplify `date-time` to just the `date`.

**Comment by sverhoeven** [comment]
Refs #622
